content/en/docs/wiki/Deprecate-and-remove-components-from-Open-Y.md

```diff
@@ -5,11 +5,25 @@ title: Deprecate and remove components from YMCA Website Services
-1. Code should be decoupled to external GitHub repositories with all history of commits, marked as `openy-decoupled`, and archived.
+2. Code should be decoupled to external GitHub repositories with all history of commits, marked as `openy-decoupled`, and archived.
+
+
+## UX/CX for deprecated components
+
+In order to deliver high quality upgrade path and keep YMCA Website Services distribution on a bleeding edge of technologies we were/are/will be removing old and aged components in favor of introducing decent for better User eXperience and Content eXperience of our community.
+
+In order to achive deprecations we have a policy which aims to serve comfortable migration path for all pieces of distribution.
+
+1. When we create component, which will replace old one in a distribution we must introduce overlap, when both components available in a system for some time ( 6-18 months usually ) in order for users have time and resources to switch anbd migrate from old to new one, before removing it from the distribution. Example - Activity Finder v4 and v3.
+2. Deprecated components should be moved to deprected module's group in Admin Extend dashboard. Also we need to add lifecycle status and information url to documentation in every deprecated module in order to provide enough information for the community. Example https://git.drupalcode.org/project/groupexpro/-/commit/67e7234257abe5f9107f9f5d4a0736e044879dc7
+3. All titles of the deprecated component in Content Editing interface should be renamed to add suffix `(deprecated)` in order to help Content Managers on a daily basis to not chose old component and use a new one. [Naming Example](<../../../../../assets/img/deprecated_naming.png>)
+4. For majority content components automated migration path is expencive and sometimes even impossible, so we have "lazy migration" practice in our community which puts responsibility for migration on shoulders of Content Managers and Strategists. Once new components is available in distribution all editors should start using it and rebuild old pages by replacing old components with the new. After communicated timeframe ( 6-18 months ) old component is removed from the distribution, but if association needs it, component will be available as a independent, but unsupported by Core team project. It can be supported by 3rd party agency or developer until it would be needed.
+5. After communicated timeframe ( 6-18 months ) Core team will remove component from distribution and keep it in independent project for archival reasons. Usually project marked as archived/obsolete in order to give clear message it is not supported and most likely unsecure.
+6. If normal timeframe ( 6-18 months ) is not achievable because of force majour reasons, Core team adds proper notifications and tutorials for the community to be able to migrate in a comfortable way in a shorter period of time. Example https://ds-docs.y.org/docs/development/daxko/#groupex-pro-apis
```

Comment:
This is very good content, just rewriting for clarity and cleanup.

```suggestion
## UX/CX for deprecated components

In order to deliver a high-quality upgrade path and keep the distribution on the bleeding edge of technologies we occasionally replace old and aged components with new ones for a better User eXperience and Content eXperience.

In order to achieve deprecations we have a policy that aims to provide a comfortable migration path for all components of the distribution.

1. When we create a component that will replace an old one we must introduce a period of overlap, when both components are available in the system for some time (6-18 months usually). This allows users to have time and resources to migrate from the old to the new one before it is removed from the distribution. See the [Activity Finder v3 to v4 migration](https://github.com/YCloudYUSA/yusaopeny_activity_finder#update-from-version-3x-to-version-4x).
2. Deprecated components are moved to the deprecated modules group in the list of modules at Admin > Extend. Also, we add [`lifecycle` and `lifecycle_link`](https://www.drupal.org/node/3215042) to the documentation in every deprecated module in order to provide enough information for the community. See [Deprecate openy_gxp.info.yml](https://git.drupalcode.org/project/groupexpro/-/commit/67e7234257abe5f9107f9f5d4a0736e044879dc7).
3. All titles of deprecated components in the Content Editing interface should be renamed to add the suffix `(deprecated)` to help Content Managers on a daily basis to not chose an old component and use a new one. ![Naming Example](<../../../../../assets/img/deprecated_naming.png>)
4. For the majority of content components an automated migration path is expensive and sometimes even impossible, so we have a "lazy migration" practice in our community which puts the responsibility of migration on Content Managers and Strategists. Once new components are available in the distribution all editors should start using them and rebuild old pages by replacing old components with new ones. After the communicated timeframe (6-18 months) old components are removed from the distribution, but if an association needs it the component will be available as an independent—but unsupported—project. It can be supported by a 3rd-party agency or developer as long as it is needed.
5. After the communicated timeframe (6-18 months) the Core team will remove the component from the distribution and keep it in an independent project for archival reasons. Usually, the project is marked as archived/obsolete in order to clarify that it is not supported and is possibly insecure.
6. If the normal timeframe (6-18 months) is not achievable due to unforeseeable circumstances, the Core team will add proper notifications and tutorials for the community to help migrate in a comfortable way in a shorter period of time. See the [GroupEx Pro API deprecation notice](https://ds-docs.y.org/docs/development/daxko/#groupex-pro-apis).
```